src/components/DailyStatistics/TransactionHistoryCard.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { FunctionComponent } from "react";
+import { size, fontSize, color } from "../../common/styles";
+import { StyleSheet, View, ActivityIndicator } from "react-native";
+import { AppText } from "../Layout/AppText";
+import { Card } from "../Layout/Card";
+
+interface TransactionHistoryCardComponent {
+  transactionHistory: {
+    name: string;
+    category: string;
+    quantityText: string;
```

Comment:
Haha curious, is there a reason we call it `quantityText` instead of `quantity` haha

Reply:
yes! because we call the util `formatQuantityText` which will format quantity with its units, + `toIsoString` as well, so it will be a string that looks like `9,999 packs` for e.g.

Reply:
ahhh, ok! haha~ thanks for explaining! ^^